app/src/main/res/values/strings.xml

```diff
@@ -64,6 +64,9 @@
     <string name="snackbar_no_pdf_app">No app to read PDF File</string>
     <string name="open_file">Open File</string>
     <string name="sort">Sort</string>
+    <string name="file_already_exists">Files with the same name already exists. Do you want to overwrite the existing file?\nPress No, to change the name.</string>
```

Comment:
Change `Files` to `File`.

Remove `\nPress No, to change the name.`. It is kind of creating a confusion.